prisma/schema.prisma

```diff
@@ -10,3 +10,54 @@ datasource db {
   url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
+
+model user {
+  id       Int        @id @default(autoincrement())
+  name     String     @db.VarChar(255)
+  profile  profile?
+  project  project[]
+  donation donation[]
+}
+
+model profile {
```

Comment:
All of these models should have an uppercase first letter to follow the prisma standards